libs/cart/src/drivers/magento/queries/remove-coupon.ts

```diff
@@ -0,0 +1,42 @@
+import gql from 'graphql-tag';
+
+import { cartItemFragment } from './fragments/cart-item';
+import { moneyFragment } from './fragments/money';
+
+export const removeAllCoupons = gql`
+  mutation RemoveAllCoupons($cartId: String!) {
+    mutation {
+      removeCouponFromCart(
+        input: {
+					cart_id: $cartId
+        }
+      ) {
+				cart {
+					id
+					items {
+						...cartItem
+					}
+					applied_coupons {
+						code
+					}
+					prices {
+						grand_total {
+							...money
+						}
+						subtotal_excluding_tax {
+							...money
+						}
+						subtotal_including_tax {
+							...money
+						}
+						subtotal_with_discount_excluding_tax {
+							...money
+						}
```

Comment:
Do we want to get `discounts` here in addition to the other prices? Not sure about it but we might need it for something.

Reply:
I think this should be done in the cart-items fragment probably. Different PR if we need it though. It causes things to break and stuff

Reply:
okay a different PR sounds good